content/chainguard/chainguard-images/shared-responsibility-model/index.md

```diff
@@ -0,0 +1,167 @@
+|-----------|:------------------------------------:|:-----------------------------------------------------:|:-------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------:|:---------------------------:|
+|  | Read-write, mutuable root filesystem | Minimal, basic human interaction inside the container | Extended, general operating system level human interaction inside the container | Outbound network connection initiation capabilities as well as package managers | Unaddressed vulnerabilities |
+|  **CHP5** |            🔒<br>read only            |                     🔒<br>no shell                     |                             🔒<br>no system utilities                            |                             🔒<br>limited networking                             |      🔒<br>low to 0 CVEs     |
+|  **CHP4** |      Read-write local filesystem     |                     🔒<br>no shell                     |                             🔒<br>no system utilities                            |                             🔒<br>limited networking                             |      🔒<br>low to 0 CVEs     |
+|  **CHP3** |      Read-write local filesystem     |                      Shell access                     |                             🔒<br>no system utilities                            |                             🔒<br>limited networking                             |      🔒<br>low to 0 CVEs     |
+|  **CHP2** |      Read-write local filesystem     |                      Shell access                     |                              System level utilities                             |                             🔒<br>limited networking                             |      🔒<br>low to 0 CVEs     |
+|  **CHP1** |      Read-write local filesystem     |                      Shell access                     |                              System level utilities                             |                 Outbound network (apk, git, wget, curl) allowed                 |      🔒<br>low to 0 CVEs     |
+|  **CHP0** |      Read-write local filesystem     |                      Shell access                     |                              System level utilities                             |                 Outbound network (apk, git, wget, curl) allowed                 |      accumulating CVEs      |
+
+Chainguard Images that meet CHP levels 0 or 1 can be used as base images to build on top of. Images meeting CHP levels 2, 3, and 4 are to be used as application-specific images. Images that align with the requirements for CHP level 5 are truly distroless application-specific images. 
```

Comment:
> Chainguard Images that meet CHP levels 0 or 1 can be used as base images to build on top of.

This implies the others can't, so I think it should be reworded.

Maybe: "CHP levels 0 and 1 are typically found in base images used in intermediate build steps."?

Reply:
>Images meeting CHP levels 2, 3, and 4 are to be used as application-specific images. Images that align with the requirements for CHP level 5 are truly distroless application-specific images.

This is a bit confusing. "are to be used"?

Reply:
Also what does "accumulating CVEs" mean? An image that intentionally isn't being updated? I think that could be reworded, maybe "(not) continuously updated"?